### ISSUE TITLE
could not find a release for this OS and architecture on gvproxy install for macos/intel

### ISSUE THREAD

**Issue by krisan** [houseabsolute/ubi#48]
**I can't install succesfully `gvproxy` tool using the `ubi`**

1. Step used for install:
```bash
ubi -dp containers/gvisor-tap-vsock -e gvproxy -t v0.7.1
```
2. Expected behaviour - force to install that unique matched version (as `-m ....` is not requried in this case).
3. Logs:
```
[ubi::ubi][DEBUG] Parsed --project containers/gvisor-tap-vsock = containers / gvisor-tap-vsock
[ubi::ubi][DEBUG] exe name = gvproxy
[ubi::ubi][DEBUG] matching = 
[ubi::ubi][DEBUG] install path = /bin/gvproxy
[reqwest::connect][DEBUG] starting new connection: https://api.github.com/
[ubi::ubi][DEBUG] current OS = macos
[ubi::ubi][DEBUG] matching assets against OS using (?i:(?:\b|_)(?:darwin|macos|osx)(?:\b|_))
[ubi::ubi][DEBUG] current CPU architecture = x86_64
[ubi::ubi][DEBUG] matching assets against CPU architecture using (?ix)
        (?:
            \b
            |
            _
        )
        (?:
            386 | i586 | i686
            |
            x86[_-]32
            |
            x86[_-]64
            |
            x64
            |
            amd64
            |
            linux64
            |
            # This is gross but the OS matcher will reject this on non-Windows
            # platforms.
            win64
        )
        (?:
            \b
            |
            _
        )

[ubi::ubi][DEBUG] matching against asset name = gvforwarder
[ubi::ubi][DEBUG] does not match our OS
[ubi::ubi][DEBUG] matching against asset name = gvproxy-darwin
[ubi::ubi][DEBUG] matches our OS
[ubi::ubi][DEBUG] does not match our CPU architecture
[ubi::ubi][DEBUG] matching against asset name = gvproxy-freebsd-amd64
[ubi::ubi][DEBUG] does not match our OS
[ubi::ubi][DEBUG] matching against asset name = gvproxy-linux-amd64
[ubi::ubi][DEBUG] does not match our OS
[ubi::ubi][DEBUG] matching against asset name = gvproxy-linux-arm64
[ubi::ubi][DEBUG] does not match our OS
[ubi::ubi][DEBUG] matching against asset name = gvproxy-windows.exe
[ubi::ubi][DEBUG] does not match our OS
[ubi::ubi][DEBUG] matching against asset name = gvproxy-windowsgui.exe
[ubi::ubi][DEBUG] does not match our OS
[ubi::ubi][DEBUG] matching against asset name = qemu-wrapper
[ubi::ubi][DEBUG] does not match our OS
[ubi::ubi][DEBUG] matching against asset name = win-sshproxy.exe
[ubi::ubi][DEBUG] does not match our OS
[ubi][ERROR] could not find a release for this OS and architecture (x86_64-apple-darwin) from gvforwarder, gvproxy-darwin, gvproxy-freebsd-amd64, gvproxy-linux-amd64, gvproxy-linux-arm64, gvproxy-windows.exe, gvproxy-windowsgui.exe, qemu-wrapper, win-sshproxy.exe
```
4. My env details:
- `ubi 0.0.28`
- MacOS Intel
   ```shell
   uname -mops
   Darwin x86_64 i386
   ```

**Comment by autarch** [comment]
Thanks for this report, and especially for including the `--debug` output. That's very helpful. I'll take a look and see if I can make this work.

**Comment by autarch** [comment]
This should be fixed in v0.0.29 release.

**Comment by krisan** [comment]
> This should be fixed in v0.0.29 release.

Yes, verified, fixed!
Thanks, @autarch
